### PR TITLE
Align getting-started.rst with the basic example

### PR DESCRIPTION
**Pull request by wachulski**
Adding missing information contained in `examples/basic`.

Saves time especially for beginners that start with the docs and assumes the page contains a complete list of steps to do.

### CODE REVIEW

**Review by marekkirejczyk — APPROVED** [review]

**Comment by marekkirejczyk** [comment]
This is a good point, but note, that it is only require if you use typescript in your project. We should add a note about it.

**Comment by marekkirejczyk** [comment]
Good catch, it is only the case if one is using TypeScript, we should add such a notes.

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -165,6 +165,8 @@ Run:
 
       npm install --save-dev mocha chai
 
+You may also need to add :code:`ts-node` and :code:`typescript` if not installed globally.
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:168]: Should be something more along these lines:
If you are using Typescirpt don't forget to add `ts-node` to your `devDependencies`

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -235,6 +237,15 @@ Update your :code:`package.json` file to include:
     }
   }
 
+Add :code:`mocha.opts` file in your test folder:
```
Comment (marekkirejczyk) [inline docs/source/getting-started.rst:240]: If you are using TypeScript add ....

File: docs/source/getting-started.rst (hunk 1 above)
Comment (wachulski) [reply docs/source/getting-started.rst]: Fixed. My intent here was to make it self-contained for a newcomer (i.e. going through it step by step literally should produce the described results). Even sth may be obvious to the more experienced.

File: docs/source/getting-started.rst (hunk 2 above)
Comment (wachulski) [reply docs/source/getting-started.rst]: Done

**Review by marekkirejczyk — APPROVED** [review]

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**